### PR TITLE
this is going great

### PR DESCRIPTION
**Pull request by daksha97**

### CODE REVIEW

File: myFillllleee.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+myFunc(xyz):
+	return xyz * funn(xzy - 1)
```
Comment (EleanorWhite) [inline myFillllleee.py]: Have you considered making this better?

Comment (EleanorWhite) [reply myFillllleee.py]: I think you could create a ore informative variable name that "xyz"

Comment (daksha97) [reply myFillllleee.py]: Can you even spell?

Comment (chenphmc) [inline myFillllleee.py]: what is funn?

Comment (chenphmc) [reply myFillllleee.py]: also which lang is this in?

Comment (chenphmc) [reply myFillllleee.py]: maybe try re enrolling in cs5, that might help ;)

Comment (daksha97) [reply myFillllleee.py]: this is python

Comment (daksha97) [reply myFillllleee.py]: also, i mispelled myFunc, stop attacking me

File: myFillllleee.py [hunk 2]
```diff
@@ -1,2 +1,2 @@
-myFunc(xyz):
-	return xyz * funn(xzy - 1)
\ No newline at end of file
+myFunc(n):
+	return n * funn(n - 1)
```
Comment (chenphmc) [inline myFillllleee.py]: right so, if this is python your func name should include "def "

Comment (chenphmc) [reply myFillllleee.py]: docstring buddy

Comment (chenphmc) [reply myFillllleee.py]: how about adding a base case?

Comment (chenphmc) [reply myFillllleee.py]: also, if this is recursion, is "funn" supposed to be "myFunc" ?

**Pull request merged**